public/ng-views/maps.html

```diff
@@ -681,11 +842,21 @@ <h4 ng-click="combosView=!combosView">
         <span class="badge" ng-show="!scenarioCompare" ng-click="scenarioCompare=true">
           Toggle Compare <i class="fa fa-toggle-off"></i>
         </span>
+        <span class="badge pull-right" ng-click="saveCombo=!saveCombo">
+          <i class="fa fa-cloud-upload"> </i> Create Scenario
+        </span>
+
+      </span>
+        <span ng-show="scenarioCompare">
+        Click right <i class="fa fa-circle-o"></i> to compare. Leave all blank for baseline.
       </span>
-    </h5>
+      <span ng-show="!scenarioCompare">
+        Select a scenario to perform a single point analysis.
+      </span>
+    </h6>
 
     <table class="table table-condensed table-hover">
-      <thead>
+      <thead style="font-size: .8em">
         <tr>
           <th><i class="fa fa-check-square"></i></th>
           <th>Name</th>
```

Comment:
Instead of using scroll, I went for the `overflow-y` to accommodate new scenarios.  Help me understand why this is a bad idea: 
![screenshot](https://cloud.githubusercontent.com/assets/7353899/8779940/fccf8538-2ed3-11e5-8b35-0761dfb22a6f.png)